assets/admin/tour/data/store.js

```diff
@@ -80,11 +81,9 @@ export const reducers = {
 	DEFAULT: ( action, state ) => state,
 };
 
-export const store = createReduxStore( SENSEI_TOUR_STORE, {
+export const SENSEI_TOUR_STORE = createStore( 'sensei/tour', {
```

Comment:
The export changed here from the store name to the store descriptor (the return of `createReduxStore`). I checked that it's only used for the selects and dispatches, and the name is the old way to use them.

So this change also fixes this. And the same applies for other occurrences in this PR.